extensions/functions_comparison.yaml

```diff
@@ -89,51 +143,68 @@ scalar_functions:
     impls:
       - args:
           - value: any1
+            name: x
         return: BOOLEAN
         nullability: DECLARED_OUTPUT
   -
     name: "is_nan"
-    description: Whether a value is not a number.  If input is null, output will be null.
+    description: >
+      Whether a value is not a number.
+
+      If `x` is `null`, `null` is returned.
     impls:
       - args:
           - value: fp32
-        nullability: MIRROR
+            name: x
         return: BOOLEAN
       - args:
           - value: fp64
-        nullability: MIRROR
+            name: x
         return: BOOLEAN
   -
     name: "is_finite"
-    description: Whether a value is finite (neither infinite nor NaN). If input is null, output will be null.
+    description: >
+      Whether a value is finite (neither infinite nor NaN).
+
+      If `x` is `null`, `null` is returned.
     impls:
       - args:
           - value: fp32
-        nullability: MIRROR
+            name: x
+        nullability: DECLARED_OUTPUT
         return: BOOLEAN
       - args:
           - value: fp64
-        nullability: MIRROR
+            name: x
+        nullability: DECLARED_OUTPUT
         return: BOOLEAN
   -
     name: "is_infinite"
-    description: Whether a value is infinite. If input is null, output will be null.
+    description: >
+      Whether a value is infinite.
+
+      If `x` is `null`, `null` is returned.
     impls:
       - args:
           - value: fp32
-        nullability: MIRROR
+            name: x
+        nullability: DECLARED_OUTPUT
         return: BOOLEAN
       - args:
           - value: fp64
-        nullability: MIRROR
+            name: x
+        nullability: DECLARED_OUTPUT
         return: BOOLEAN
   -
     name: "nullif"
     description: If two values are equal, return null. Otherwise, return the first value.
     impls:
       - args:
           - value: any1
+            name: x
           - value: any1
+            name: y
+        nullability: DECLARED_OUTPUT
```

Comment:
This doesn't seem right either, can't return null as per the description if both inputs are non-nullable and equal.